schnelleinsteig.test.js

```diff
@@ -15,145 +15,41 @@ const rondelSlots = [
 ];
 
 describe("Schnelleinsteig", () => {
-  describe("setup for four players", () => {
-    test("All players receive 13 million", () => {
-      const log = [
-        Action.playerSeating({ order: ["Daniel", "Claudia", "Bert", "Anton"] }),
-      ];
+  describe.only("setup for four players", () => {
```

Comment:
Removing this makes all the Schnelleinsteig tests fail 😿 

I think I'll remove all the changes I made to Schnelleinsteig, especially since I don't want to step on your toes.